docs/reference/release-notes/7.0.0-rc1.asciidoc

```diff
@@ -1,6 +1,203 @@
 [[release-notes-7.0.0-rc1]]
 == {es} version 7.0.0-rc1
 
-Also see <<breaking-changes-7.0,Breaking changes in 7.0>>.
+coming[7.0.0-rc1]
```

Comment:
I'm not sure why we have this line.

Reply:
That'll get removed when it goes live tomorrow

Reply:
Thanks @lcawl